lib/board.rb

```diff
@@ -11,4 +11,23 @@ def initialize
       [nil, nil, nil]
     ]
   end
+
+  def make_move(piece, position)
+    row_and_col = position_to_row_and_col position
```

Comment:
Could this be simpler? Whenever I have to use the word 'and' I think of it as a sign that my method/class/module should be broken apart.